src/coreclr/vm/codeman.h

```diff
@@ -2594,6 +2910,37 @@ ULONG       GetFixedStackSize();
 
 };
 
+#ifdef FEATURE_INTERPRETER
+
+inline InterpreterCodeHeader* InterpreterJitManager::GetCodeHeader(const METHODTOKEN& MethodToken)
+{
+    LIMITED_METHOD_DAC_CONTRACT;
+    _ASSERTE(!MethodToken.IsNull());
+    return dac_cast<PTR_InterpreterCodeHeader>(MethodToken.m_pCodeHeader);
+}
+
+inline InterpreterCodeHeader * InterpreterJitManager::GetCodeHeaderFromStartAddress(TADDR methodStartAddress)
+{
+    LIMITED_METHOD_DAC_CONTRACT;
+    _ASSERTE(methodStartAddress != (TADDR)NULL);
+    ARM_ONLY(_ASSERTE((methodStartAddress & THUMB_CODE) == 0));
```

Comment:
nitpick: This looks copy-pasted from the JIT version and could bite us if we ever make interpreter opcodes single-byte.
Alternately, we could explicitly enforce the alignment of interpreter IR here (i think it's 4 bytes right now?)

Reply:
Right, this assert should not be here.